modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/config/DefaultMutableConfigurationStats.java

```diff
@@ -89,6 +100,7 @@ public int discountInflightOperation() {
   @Override
   public int addRunningSource() {
```

Comment:
I would implement these deprecated methods as, for example:

addActiveComponent();
return runningSources.incrementAndGet();

Reply:
sure